src/platform/plugins/shared/discover/public/context_awareness/profile_providers/observability/traces_data_source_profile/profile.test.ts

```diff
@@ -36,6 +36,18 @@ describe('tracesDataSourceProfileProvider', () => {
         } as unknown as DataView,
       } as DataSourceProfileProviderParams)
     ).toEqual(RESOLUTION_MATCH);
+
+    expect(
+      tracesDataSourceProfileProvider.resolve({
+        dataSource: {
+          type: DataSourceType.DataView,
+          dataViewId: 'apm_static_data_view_id_custom_view',
+        },
```

Comment:
```suggestion
        dataSource: createDataViewDataSource({ dataViewId: 'apm_static_data_view_id_custom_view' }),
```
Nit: It looks like this is just following an existing pattern in the file, but figured I should mention we also have a util for this that we usually use.